docs/basics/101-121-siblings.rst

```diff
@@ -218,37 +218,37 @@ the changed or added files from the sibling inside, and put it on your
 desk. You can now take a look into that drawer to see whether you want
 to have the changes right in front of you.
 
-The drawer is a branch, and it is usually called ``remotes/origin/master``.
+The drawer is a branch, and it is usually called ``remotes/origin/main``.
 To look inside of it you can :command:`git checkout BRANCHNAME`, or you can
 do a ``diff`` between the branch (your drawer) and the dataset as it
 is currently in front of you (your desk). We will do the latter, and leave
 the former for a different lecture:
 
-.. windows-wit:: Please use datalad diff --from master --to remotes/roommate/master
+.. windows-wit:: Please use datalad diff --from main --to remotes/roommate/main
 
    Please use the following command instead::
 
-      datalad diff --from master --to remotes/roommate/master
+      datalad diff --from main --to remotes/roommate/main
 
-   This syntax specifies the :term:`master` :term:`branch` as a starting point for the comparison instead of the current ``adjusted/master(unlocked)`` branch.
+   This syntax specifies the :term:`main` :term:`branch` as a starting point for the comparison instead of the current ``adjusted/main(unlocked)`` branch.
```

Comment:
```suggestion
   This syntax specifies the :term:`main` :term:`branch` as a starting point for the comparison instead of the current ``adjusted/master(unlocked)`` branch.
```